edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/Authenticator.cs

```diff
@@ -1,20 +1,25 @@
+// <copyright file="Authenticator.cs" company="Microsoft">
 // Copyright (c) Microsoft. All rights reserved.
-
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// </copyright>
 namespace Microsoft.Azure.Devices.Edge.Hub.Core
 {
-    using System;
     using System.Threading.Tasks;
+
     using Microsoft.Azure.Devices.Edge.Hub.Core.Device;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
     using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Extensions.Logging;
+
     using static System.FormattableString;
 
     public class Authenticator : IAuthenticator
     {
-        readonly string edgeDeviceId;
-        readonly IAuthenticator tokenAuthenticator;
-        readonly ICredentialsCache credentialsCache;
+        private readonly ICredentialsCache credentialsCache;
```

Comment:
I think we should discuss this.. not having private on fields was done intentionally throughout our code.. I like it and I am hesitant to change it.

Reply:
I am very open; I found ppl discussed in Stackoverflow; https://stackoverflow.com/questions/254912/should-you-use-the-private-access-modifier-if-its-redundant.  I am ok to not having private.